app/2.0/docs/tools/deploy.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Deploy your app
```

Comment:
I think the title is probably off here, because the main subject of this doc is the PRPL pattern and the app shell structure that our build tools expect.

The "Build output" section is really discussing how the build chain bundles an app built with the PRPL app shell pattern.

Title suggestions? The PRPL pattern? Copying @kevinpschaaf @graynorton